WordPress/gutenberg
# Restore `@wordpress/nux` to WP 6.3 branch

## What?
Reverts #46110 in the WordPress 6.3 branch. See #52444.

See also:
* #52452 -- Equivalent for WP 6.2 branch
* #52455 -- Equivalent for trunk

> **Warning**
>
> See the original ticket for a [warning about getting the commit and package release order correct.](https://github.com/WordPress/gutenberg/issues/52444)


> **Note**
>
> Depending on how you wish to version the restored nux package, this may not be required if the package can be backported from trunk.


## Why?

As the WordPress-Develop is still consuming the package, this is causing significant issues while running npm run sync-gutenberg-packages for both the wp-6.2 and wp-6.3 distribution tags.

While the script is apparently successful, the incorrect versions of key packages are installed. @wordpress/components reverts to version 23.9.0; @wordpress/data reverts to 8.6.0.

Tagging @wordpress/nux@6.0.0 with the affected distribution tags has been attempted without success, see [WP#57643 (comment)](https://core.trac.wordpress.org/ticket/57643#comment:29).


## How?


* Reverts 9ea80f6f363f847b31c5faa3bac6be1c05f5ef63
* Adding myself as a code owner on the basis that if I revert, I own it.

## Testing Instructions

1. Run `npm i`
2. Run `git checkout package-lock.json` if required
3. Run `npm distclean; npm clean:packages`
4. Ensure the build files in `packages/nux` don't exist
5. Run `npm build`
6. Ensure the build files in `packages/nux` have been created
7. Run `npm run docs:build`
8. Ensure no files are changed and the doc changes are commited

**Lerna dry run: TBA**

### Testing Instructions for Keyboard

N/A -- code change only

## Screenshots or screencast
